README.md

```diff
@@ -1,17 +1,27 @@
-# cloud-platform-terraform-_template_
+# cloud-platform-terraform-secret-manager
 
-_note: Please remove all comments in italics and fill where required>_
-
-_Please change the urls in the release badge below_
-[![Releases](https://img.shields.io/github/release/ministryofjustice/cloud-platform-terraform-template/all.svg?style=flat-square)](https://github.com/ministryofjustice/cloud-platform-terraform-template/releases)
-
-_Short describion of the module_
-_This Terraform module ......_
+This Terraform module is used to create and manage secret manager resources in the AWS.
 
 ## Usage
 
-_Describe how to use the module_
-See the [examples/](examples/) folder.
+```
+module "secrets_manager" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-secrets-manager?ref=concourse"
```

Comment:
Can you make this a tag (doesn't have to be real)?

Reply:
Done